pkg/actions/cluster/get.go

```diff
@@ -58,7 +59,7 @@ func GetClusters(provider api.ClusterProvider, listAllRegions bool, chunkSize in
 			continue
 		}
 		// Reset region and recreate the client.
-		ctl, err := newClusterProvider(&api.ProviderConfig{
+		ctl, err := newClusterProvider(context.TODO(), &api.ProviderConfig{
```

Comment:
shouldn't this be passed in here too?

Reply:
It's a serious question. :D Like, is it `GetClusters`? Or is it `getAndPrinterClusters` or is it `doGetCluster` which is added in 

```go
	cmd.CobraCommand.RunE = func(_ *cobra.Command, args []string) error {
		cmd.NameArg = cmdutils.GetNameArg(args)
		return doGetCluster(cmd, params, listAllRegions)
	}
```

That's added from `getClusterCmd` that's added in `Command` which is initalised in main. :)

Reply:
I'd stop at ctl

Reply:
Ok, moved it out until ctl.